about.html

```diff
@@ -0,0 +1,164 @@
+          <nav class="nav-bar-desktop">
+            <a href="index.html"><img src="images/logo.png" alt="" /></a>
+            <div class="desktop-links">
+              <ul>
+                <li class="list-margin">
+                  <a href="about.html" id="actual-menu">About</a>
+                </li>
+                <li class="list-margin"><a href="#">Program</a></li>
+                <li class="list-margin"><a href="#">Join</a></li>
+                <li class="list-margin"><a href="#">Sponsor</a></li>
+                <li class="list-margin"><a href="#">News</a></li>
+                <li><a href="" id="last-menu">Crypto & New Order</a></li>
+              </ul>
+            </div>
+          </nav>
+        </div>
+      </header>
+      <section class="intro-about">
+        <div class="intro-title-about">
+          <h1>CONVECTION ABOUT CRIPTOS AND THE NEW ORDER</h1>
+        </div>
+        <div class="intro-text-about">
+          <p>
+            Our first big conference in Venezuela to talk about cryptos with the
+            best of the best! Thanks to our partner agencies around the globe,
+            we are able to make the most big conference in latin America. They
+            say they critpcurrencies are the future. They assure us they want to
+            help us become a better civilization. They teach us about the new
+            tecnologies to come. Are you ready? Sign up and prepare to the
+            future!
+          </p>
+        </div>
+        <div class="intro-contact">
+          <p>
+            Please contact us per Email for any further questions about our
+            Criptocurrencie conference!
+          </p>
+          <p class="contact-email"><strong>criptosfuture@gmail.com</strong></p>
+        </div>
+      </section>
+
+      <section class="logo-about">
+        <div class="title-logo">
+          <h2>Crypto and the New Order logo 2022</h2>
+          <div class="orange-line"></div>
+          <p>
+            The logo for the convection "CRYPTO AND THE NEW ORDER" was decided
+            from a group of the finest dessigners
+          </p>
+        </div>
+        <div class="logo-container">
+          <img src="images/logo.png" alt="Logo crypto and the new order" />
+        </div>
+      </section>
+
+      <section class="past-convention">
+        <div class="past-title">
+          <h2>See the past Crypto and New Order conventions</h2>
+          <div class="orange-line"></div>
+          <p>
+            Take a look to the last two Crypto and New Order wich took place in
+            Spain and Argentina
+          </p>
+        </div>
+        <div class="past-convention-card">
+          <div class="spain card-past">
+            <h3>2010</h3>
+            <h4>Crypto and New Order Barcelona Spain</h4>
+          </div>
+          <div class="argentina card-past">
+            <h3>2018</h3>
+            <h4>Crypto and New Order Buenos Aires Argentina</h4>
+          </div>
+        </div>
```

Comment:
- [ ] Please decrease the gap between these cards and make sure it matches the design on the template

### Your design

![image](https://user-images.githubusercontent.com/83803414/163426791-729db60d-093e-43d4-afd4-f44402ecf631.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/163426825-43368d3f-1b3e-4be0-a515-f4f20421ebe9.png)